docs/howto/run_material_mapping.md

```diff
@@ -0,0 +1,119 @@
+
+
+```
+python3 ../acts-framework/scripts/MaterialMaping/writeMapConfig.py geometry-map.json config-map.json
+```
+
+Then edit the config-map.json file
+
+```
+../acts-framework/scripts/MaterialMaping/configureMap.py geometry-map.json config-map.json
+```
+
+## Geantino scan
+
+The next step is to do a geantino scan of our detector. For this we will use the GeantinoRecording algorithm :
+
+```
+./../build/bin/ActsSimGeantinoRecording -j1 --dd4hep-input ../Examples/Detectors/DD4hepDetector/compact/OpenDataDetector/OpenDataDetector.xml --output-root true -n10000
+```
+
+The result of the geantino scan will be a root file containing Material Tracks, those contain the direction and production vertex of the geantino, the total material accumulated and all the interaction point in the detector.
+
+## Material Mapping
+
+With the surfaces map and the material track we can finally do the material mapping using the MaterialMapping algorithm :
+
+```
+./../build/bin/ActsExampleMaterialMappingDD4hep -j1 --input-root true --input-files geant-material-tracks.root --mat-input-type file --mat-input-file geometry-map.json --output-root true --output-json true --mat-mapping-collection material-tracks --mat-output-file material-maps --mat-mapping-surfaces true --mat-mapping-volumes true --mat-mapping-volume-stepsize 1 --dd4hep-input ../Examples/Detectors/DD4hepDetector/compact/OpenDataDetector/OpenDataDetector.xml
+```
+
+As an output you will obtain the material map as a root and json file and a new Material Tracks collection in a root file. This new collection add to each material interaction the associated surface during the mapping. This can be used for the control plots.
+Depending on what you want to do there are three option you can change :
+*  `mat-mapping-surfaces` : determine if material is mapped onto surfaces
+*  `mat-mapping-volumes` : determine if material is mapped onto volumes
+*  `mat-mapping-volume-stepsize` : Determine the step size used in the sampling of the volume this should be small compared to you bin size.
+
+You can map onto surfaces and volumes separately (for example if you want to optimise one then the other). In that case after mapping one of those you will need to use the resulting json material map as an input to the `mat-input-file`.
+
+## Material Validation
+
+Now that the map has been written, you will want to validate it. First you can use the MaterialValidation example. This will perform propagation throughout the detector once it has been decorated with the material map. It will then output material tracks with the same format as the one obtain with the Geantino.
```

Comment:
```suggestion
Now that the map has been written, you will want to validate it. First you can use the `MaterialValidation` example. This will perform propagation throughout the detector once it has been decorated with the material map. It will then output material tracks with the same format as the one obtain with the Geantino.
```